gbpservice/nfp/orchestrator/coal/networking/nfp_gbp_network_driver.py

```diff
@@ -42,6 +41,13 @@ def update_port(self, token, port_id, port):
                                                        port)
         return pt['port_id']
 
+    def get_neutron_port_details(self, token, port_id):
```

Comment:
This method name can be get_port_and_subnet_details. So that both nautron and gbp workflow call same method. It just looks like,

network_handler.get_port_and_subnet_details(token, port_id)

Is this port_id a neutron port ID ?. We should pass GBP pt id here in GBP workflow.

And in this method, get neutron port_id like,

_port_id = self.get_port_id(token, port_id)

Ref: get_port_details method in the same file.